### ISSUE TITLE
PiP action desktop file

### ISSUE THREAD

**Issue by donadigo** [elementary/gala#649]
It seems like we're getting a lot of "I never knew PiP existed in the system", even though we have the shortcut displayed in the shortcuts overlay, I think we could include a desktop file for initiating PiP, much like we do with multitasking view.

**Comment by cassidyjames** [comment]
I'm not super on board with adding a .desktop file for Gala features. Multitasking View is there because we want it on the dock by default, but we really would prefer it not go in Applications Menu because it's not an app. But it's there now because otherwise there's not a good way to get it back onto the dock if it's accidentally removed.

I think long term a better route would be to integrate PiP into apps themselves; manually-invoked PiP is handy but it really feels like a workaround. Being more integrated into i.e. the Videos app could help expose it as a feature, which then would help people know what it is when they see that feature in the Shortcuts Overlay.

**Comment by sprite-1** [comment]
> But it's there now because otherwise there's not a good way to get it back onto the dock if it's accidentally removed.

How about adding it to the Docklet portion of the Plank's menu?

![image](https://user-images.githubusercontent.com/33685433/70453101-27737d80-1ae3-11ea-8348-d4c5d43780ef.png)

**Comment by cassidyjames** [comment]
@sprite-1 yeah I think that's a better place, though it's still pretty hidden. But we talked about implementing it either as a hidden .desktop or an applet, then having a switch in _System Settings_ → _Desktop_ → _Dock_ to enable it maybe.

**Comment by sprite-1** [comment]
@cassidyjames I hope it's still movable in the icon order when/if you do end up making it a System Setting entry

**Comment by danrabbit** [comment]
After some discussion in Slack it sounds like we're maybe in agreement that instead of a launcher this would make more since as a menu item in the header context menu. @cassidyjames can we go ahead and confirm and update the issue title to that effect?
